crates/typst/src/text/font/book.rs

```diff
@@ -403,20 +399,7 @@ fn typographic_family(mut family: &str) -> &str {
     // Apply style suffix trimming.
     family = &family[..len];
 
-    if newcm {
-        family = family.trim_end_matches("10");
-    }
-
-    // Fix bad names.
-    match family {
-        "Noto Sans Symbols2" => "Noto Sans Symbols 2",
```

Comment:
This rename is gone (incorporating it would mean patching each variant separately though, which is a bit unfortunate)

Reply:
Noto Sans Symbols 2 corrects it's family name since [Sep, 2022](https://github.com/notofonts/symbols/releases/tag/NotoSansSymbols2-v2.005). Maybe we can drop this heuristics?

Reply:
Ah, that's nice. I'll check whether we need to update the font in the web app.

Reply:
Still, do you think a mechanism for patching a family name for a full family is necessary or the current approach of patching each POSTSCRIPT_NAME individually is the right way to go?

Reply:
I don't think adding a new mechanism specially for family names worth the effort. I prefer to keep it simple. And we need the ability to specify a family name for an individual font regardless.